services/src/main/java/io/druid/cli/CreateTables.java

```diff
@@ -66,6 +68,9 @@ public CreateTables()
   protected List<? extends Module> getModules()
   {
     return ImmutableList.<Module>of(
+        new DruidProcessingModule(),
```

Comment:
For some unknown reason yes, when I didn't include it it was failing integration tests (metadata-init is run from Docker)